cloudbank-v4/customer-helidon/pom.xml

```diff
@@ -208,6 +215,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Helidon Maven Plugin for native image support -->
+            <plugin>
+                <groupId>io.helidon.build-tools</groupId>
+                <artifactId>helidon-maven-plugin</artifactId>
+                <version>${helidon-maven-plugin.version}</version>
+            </plugin>
+            <!-- Eclipse JKube for container image building -->
+            <plugin>
+                <groupId>org.eclipse.jkube</groupId>
+                <artifactId>kubernetes-maven-plugin</artifactId>
+                <version>${jkube.version}</version>
+                <configuration>
+                    <images>
```

Comment:
this did not work for me,  instead, remove the configuration section from here and instead add a property (up above) as follows: 

```
   <properties>
        <jkube.version>1.16.2</jkube.version>
        <jkube.generator.name>${obaas.registry}/${project.artifactId}:${obaas.image.version}</jkube.generator.name>
   ...
```